EU4toV2/Data_Files/reformMapping.txt

```diff
@@ -0,0 +1,666 @@
+# Copyright (c) 2019 The Paradox Game Converters Project
```

Comment:
No need for the copyright notice in new files (feel free to remove in old files too), as we now have a LICENSE file handling it.

Reply:
dropped copyright block from all txt files.